deploy/jenkins-ci/Jenkinsfile

```diff
@@ -139,6 +149,24 @@ node('kiali-build') {
         sh "cd ${operatorDir}; git config user.name 'kiali-bot'"
         sh "cp ${backendDir}/deploy/jenkins-ci/${operatorMakefile} ${operatorDir}"
       }
+      if ( buildHelm ) {
+        checkout([
+          $class: 'GitSCM',
+          branches: [[name: 'master']],
```

Comment:
I would like to point out that notice you cannot define the branch here (there is no HELM_RELEASING_BRANCH). It must always be `master` because that is where GitHub Pages will serve its content from. So `master` is the branch where all the release artifacts must go, such as the helm chart tarballs and the helm `index.yaml`.